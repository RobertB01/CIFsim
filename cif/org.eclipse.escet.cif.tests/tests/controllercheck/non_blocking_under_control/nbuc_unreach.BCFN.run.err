WARNING: The specification contains no input variables.
ERROR: CIF controller properties checker failed due to unsatisfied preconditions:

  -------------------------------------------------------------------------------------------------------------------
  (1/1) Non-deterministic edges with overlapping guards within a location, for controllable event "Supervisor.c_act".
  -------------------------------------------------------------------------------------------------------------------
   * In automaton "Supervisor":
     - location Unreach2:
                ^
