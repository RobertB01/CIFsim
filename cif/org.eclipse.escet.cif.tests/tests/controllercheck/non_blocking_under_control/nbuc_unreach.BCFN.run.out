Loading CIF specification "controllercheck/non_blocking_under_control/nbuc_unreach.cif".
Preparing for BDD-based checks:
    CIF variables and location pointers:
        Nr     Kind              Type  Name        Group  BDD vars  CIF values  BDD values  Values used
        -----  ----------------  ----  ----------  -----  --------  ----------  ----------  -----------
        1      location pointer  n/a   Supervisor  0      3 * 2     5 * 2       8 * 2       ~63%
        -----  ----------------  ----  ----------  -----  --------  ----------  ----------  -----------
        Total                                      1      6         10          16          ~63%

    Skipping variable ordering: only one variable present.

    Restricting system behavior using state/event exclusion plant invariants:
        No guards changed.

Preparing for MDD-based checks:
